packages/storybook/src/designsystem/Typography.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react'
+import { Box, Text, Flex } from '@kodiak/primitives'
+import { theme } from '../theme'
```

Comment:
@alexanderchan should we use `useThemeUI` here instead of importing the theme directly?

Reply:
@wesleycole sounds good, required a few type checks and an updated config to get this working but it fixed the problem of context not being able to be used in the story.